server/GraknService.java

```diff
@@ -55,13 +56,13 @@
 
     private final Grakn grakn;
     private final ConcurrentMap<UUID, SessionService> sessionServices;
-    private final TransactionExecutor executor;
+    private final AsyncTransactionExecutor executor;
 
     public GraknService(Grakn grakn) {
         this.grakn = grakn;
         sessionServices = new ConcurrentHashMap<>();
-        // TODO: this is temporarily not used until we enable TransactionExecutor
-        //executor = new TransactionExecutor(PARALLELISATION_FACTOR, TRANSACTION_EXECUTOR_QUEUE_FACTOR);
+        // TODO: Enable this along with AsyncTransactionExecutor
+        //executor = new AsyncTransactionExecutor(PARALLELISATION_FACTOR, TRANSACTION_EXECUTOR_QUEUE_FACTOR);
```

Comment:
Is this related to the reintroduction of `executorServiceExecutor` above in the PR?

Reply:
I've decided to just remove the comments for now. We'll bring them back properly when we get to implement `matchAsync` and `insertAsync`.